lib/u3d/installer.rb

```diff
@@ -384,7 +397,70 @@ def exist?
     def editor_version
       require 'yaml'
       yaml = YAML.load(File.read("#{@path}/ProjectSettings/ProjectVersion.txt"))
-      yaml['m_EditorVersion']
+      version = yaml['m_EditorVersion']
+      if Helper.linux?
+        version.gsub!(/Linux/, '')
+        version.gsub!(/x/, '')
+      end
+      version
+    end
+  end
+
+  class LinuxDependencies
+    DEPENDENCIES = [
```

Comment:
I'm not sure the [blogpost](https://forum.unity3d.com/threads/unity-on-linux-release-notes-and-known-issues.350256/) presents the  dependencies alongside the first release, but the dependencies are no longer mentionned in this thread, which is supposed to be the official one.